README.md

```diff
@@ -22,4 +22,38 @@ content-type:application/json
     'string': 'abacaba',
     'float': 1337.0
 }
+```
+
+Deploying to Ubuntu with preinstalled Docker on Azure cloud
+---
+
+To do deployment on Auzre cloud you need to obtain subscription on [Azure](https://portal.azure.com)
+
+0. After [Nginx setup](https://www.techrepository.in/serving-static-files-using-nginx-in-docker) and [Nginx as reverse proxy setup](https://www.techrepository.in/using-nginx-as-a-reverse-proxy-for-asp-net-core-web-app-in-docker)
+you can make docker image by
+```
+$ cd ./server 
+$ docker build -t revproxy .
+```
+1. Request Docker on Ubuntu Server (Canonical + Microsoft) [here](https://portal.azure.com)
+Make ssh connection with server, so you can login with.
+```
+$ ssh 53.53.53.53
```

Comment:
The next two commands are executed on localhost, please arrange in a logical order